cmd/local-gadget/main.go

```diff
@@ -273,124 +274,151 @@ func newRootCmd() *cobra.Command {
 }
 
 func main() {
-	var err error
-	localGadgetManager, err = localgadgetmanager.NewManager()
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
-	}
+	var verbose bool
+
+	inlineCmd := &cobra.Command{
+		Use:          "local-gadget",
+		Short:        "Collection of gadgets for containers",
+		SilenceUsage: true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if verbose {
+				log.StandardLogger().SetLevel(log.DebugLevel)
+			}
 
-	homedir, err := os.UserHomeDir()
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
-	}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Since this function is quite long, could it be moved outside of main() and be referenced here?

Reply:
You are rigth. The new version looks much better!